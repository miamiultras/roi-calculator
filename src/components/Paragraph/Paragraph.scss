.paragraph  {
  p {
    font-size: 21px;
    max-width: 650px;
    line-height: 130%;
    margin: 28px auto;
  }
}
