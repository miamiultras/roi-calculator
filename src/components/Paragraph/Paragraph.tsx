import React from 'react';

import './Paragraph.scss';

const Paragraph = ({ children }: { children: string }) => (
  <div className='paragraph'>
    <p>{children}</p>
  </div>
);

export default Paragraph;
