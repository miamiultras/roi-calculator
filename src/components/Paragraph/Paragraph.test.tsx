import * as React from 'react';
import { shallow } from 'enzyme';
import Paragraph from './Paragraph';

function setup() {
  const props = {};
  const wrapper = shallow(<Paragraph>test</Paragraph>);
  return {
    wrapper,
    props,
  };
}

describe('Paragraph component', () => {
  it('should match snapshot if no change was intended', () => {
    const { wrapper } = setup();

    expect(wrapper).toMatchSnapshot();
  });
});
