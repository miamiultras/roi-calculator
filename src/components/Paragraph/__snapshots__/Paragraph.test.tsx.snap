// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph component should match snapshot if no change was intended 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Paragraph>
    test
  </Paragraph>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        test
      </p>,
      "className": "paragraph",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "test",
      },
      "ref": null,
      "rendered": "test",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          test
        </p>,
        "className": "paragraph",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "test",
        },
        "ref": null,
        "rendered": "test",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
